frontend/packages/integration-tests-cypress/tests/crud/k8-openshift-cruds.spec.ts

```diff
@@ -24,55 +24,72 @@ describe('Kubernetes resource CRUD operations', () => {
     cy.logout();
   });
 
+  const tableColumnHeaders =
+    'th > .pf-c-table__button > .pf-c-table__button-content > .pf-c-table__text';
```

Comment:
This selector looks pretty fragile. Is there anything simpler we can do?

Reply:
fixed to just `th .pf-c-table__text`